traffic_portal/test/integration/PageObjects/CacheGroup.po.ts

```diff
@@ -158,6 +156,26 @@ export class CacheGroupPage extends BasePage {
         await snp.NavigateToCacheGroupsPage();
         return result;
     }
+    public async CheckCSV(name: string): Promise<boolean> {
+        return element(by.cssContainingText("span", name)).isPresent();
+    }
+    public async ToggleTableColumn(name: string): Promise<boolean> {
+        let result = false;
+        await this.btnTableColumn.click();
+        //if it's already on, turn it off and return false
+        if (await browser.isElementPresent(element(by.xpath("//th[text()='" + name + "']"))) === true){
+            await element(by.xpath("//label[text()=' " + name + "']")).click();
+            result = false;
+        }else{
+            //of it already off, turn it on return true
+            await element(by.xpath("//label[text()=' " + name + "']")).click();
```

Comment:
It seems like you do this click on the same element regardless of the `if` condition, could it be moved out of the `if`/`else` entirely? If so, this method can be reduced to simply:

```typescript
public async ToggleTableColumn(name: string): Promise<boolean> {
    await this.btnTableColumn.click();
    const result = await element(by.cssContainingText("th", name).isPresent();
    await element(by.cssContainingText("label", name)).click();
    await this.btnTableColumn.click();
    return !result;
}
```
no `if` required